SETUP.md

```diff
@@ -20,10 +20,27 @@ NOTE: This will connect with our ownCloud iOS Library repository at "https://git
   - Init the library submodule: "git submodule init"
   - Update the library submodule: "git submodule update"
 
-### 3. Create pull request:
+### 3. Create your own certificates
+* Login at https://developer.apple.com/ as developer and there to to the Certificates section.
+* Create a Development Certificate for you (probably you got it one now)
+* Create an App Id for the main app. Ex: com.mywebpage.owncloud.ios
```

Comment:
It could be important to say that the name of the extension bundles must be has this pattern:
ios app bundle id -> com.mywebpage.owncloud.ios
extension bundle id -> com.mywebpage.owncloud.ios.extensionname. 
If not, doesn't work.